5.machine_learning/scripts/3.visualize_model_coef.r

```diff
@@ -1,19 +1,28 @@
 suppressPackageStartupMessages(library(ggplot2))
 suppressPackageStartupMessages(library(dplyr))
 
+# insall ggmagnify from hughjonesd's universe to zoom in on plots
+install.packages("ggmagnify", repos = c("https://hughjonesd.r-universe.dev", 
```

Comment:
Consider adding this package to your environment specification. If it's inconvenient to add it everywhere, consider a localized environment specific to just this directory.

Reply:
After some investigation, it looks like it isn't installable via conda or pip so this is the only way of doing it. It is not convenient but it is a limitation of the software.